charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -18,7 +18,7 @@ sidecars:
     tag: "v3.0.0"
   snapshotterImage:
     repository: k8s.gcr.io/sig-storage/csi-snapshotter
-    tag: "v3.0.3"
+    tag: "v4.0.0"
```

Comment:
4.0 requires k8s 1.20+. I don't think we should do this just yet.

https://kubernetes-csi.github.io/docs/external-snapshotter.html

Reply:
darn, ok